src/qt/walletmodel.cpp

```diff
@@ -57,6 +52,14 @@ WalletModel::~WalletModel()
     unsubscribeFromCoreSignals();
 }
 
+void WalletModel::init()
```

Comment:
nit: Could slot name be more expressive, as it initiates regular polling?

Reply:
Yeah like `startPollBalance`?

Reply:
Yup ;)